src/client/components/add-batch-page/add-batch-page.style.css

```diff
@@ -0,0 +1,3 @@
+.add-batch {
+  margin: 0 0 0 180px;
```

Comment:
Don't really like that we need to push in the content like this, but I see we're doing the same on other pages as well. Overall the CSS layout is not done very well with both the menu and footer being position:fixed, but I don't think we should fix this in your PR and maybe we should not touch it before wednesday.

Reply:
Could you maybe add a TODO comment above this line, so we can find it later?